Trinap/Sources/Domain/UseCases/Sue/DefaultCreateSueUseCase.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  DefaultCreateSueUseCase.swift
+//  Trinap
+//
+//  Created by kimchansoo on 2022/12/07.
+//  Copyright © 2022 Trinap. All rights reserved.
+//
+
+import Foundation
+
+import RxSwift
+
+final class DefaultCreateSueUseCase: CreateSueUseCase {
+
+    // MARK: Properties
+    private let sueRepository: SueRepository
+    
+    // MARK: Initializers
+    init(sueRepository: SueRepository) {
+        self.sueRepository = sueRepository
+    }
+    
+    // MARK: Methods
+    func create(suedUserId: String, contents: Sue.SueContents) -> Observable<Void> {
+        var rawContents = ""
+        
+        switch contents {
+        case .blame:
+            rawContents = contents.contents
+        case .hateSpeech:
+            rawContents = contents.contents
+        case .annoyance:
+            rawContents = contents.contents
+        case .etc(let string):
+            rawContents = string
+        }
```

Comment:
```suggestion
        let rawContents = contents.contents
```
이거는 안될까요..?

아래 코드에서 contents면 바로 String을 반환하는 것 같아서 여쭤봅니다!

https://github.com/boostcampwm-2022/iOS02-Trinap/blob/2eebf7100530ac6adab91fb6a759e8797d009528/Trinap/Sources/Domain/Models/Sue.swift#L51-L62